### ISSUE TITLE
Show map of no-neighbourhood events

### ISSUE THREAD

**Issue by captn3m0** [blr-today/website#18]
We need to generate a geojson file that:

1. Has a list of neighbourhoods
2. Puts a dot for every neighbourhood with a small 5m randomness
3. Colors every dot according to the neighborhood
4. Marks all events without a neighbourhood with a separate color.

The idea is to look at this generated map, and decide what neighborhood we want to add or expand, and help us decide on the boundaries if we go that way to tag.

I've also noticed some events that use a 12,77 lat/lng (with no decimals). Do check where that falls, for further analsysis and fixing.

**Comment by kdqed** [comment]
@captn3m0 shall I take this up? I was thinking of putting all the events on a map. Maybe I can start with that, and then add the coloring. We might not need a geojson file to build the UI, we could use the SQLite file that's already there.

**Comment by captn3m0** [comment]
Yup, a sqlite -> regular JSON will also do. The one big problem is that translating events -> location gets rid of "multiple events at the same location" (A few dots in the same place isn't the same as a long list of events at UC for eg).

Then there are events with incorrect, low-resolution, or TBA geolocation. Do give it a try, but this is going to take some learnings before we get to a usable/cool visualization.
